eth/downloader/downloader.go

```diff
@@ -162,9 +161,6 @@ type Core interface {
 
 	// Snapshots returns the core snapshot tree to paused it during sync.
 	Snapshots() *snapshot.Tree
-
-	// GetTd returns the total difficulty of a local block.
```

Comment:
Why did you remove this from the core interface, but not remove the implementation(s)?